src/prompts/model_new_ex_add_triton.py

```diff
@@ -0,0 +1,63 @@
+import torch
```

Comment:
the name of this directory confused me a bit at first

Reply:
I agree, but its how the codebase haha